extensions/amp-fit-text/1.0/component.js

```diff
@@ -15,14 +15,12 @@
  */
 
 import * as Preact from '../../../src/preact';
-import * as styles from './component.css';
 import {ContainWrapper} from '../../../src/preact/component';
+import {LINE_HEIGHT_EM_, useStyles} from './component.jss';
```

Comment:
Not 100% the constant import would work here since the "jss" file is completely rewritten potentially. /cc @samouri

Reply:
Should be fine. The transform is relatively surgical and doesn't affect anything outside its small scope. If I remember right it just:
1. removes the `createUseStyles` import
2. Rewrites the createUseStyles CallExpression to a literal value (the string map)

Everything else is left as-is. We don't have any tests enforcing it stays that way though so I'll add one!

tldr: this is fine

Reply:
proof is in the test pudding: https://github.com/ampproject/amphtml/pull/32399

Reply:
Yum, thanks!